### PR TITLE
HTTPS - simplify bloom filters, add .bin bloom filter

### PR DESCRIPTION
**Pull request by kdzwinel**
- generate new test bloom filters that are very simple (3-6 entries)
- document them better in the readme
- include `.bin` bloom filter and corresponding `_spec` file
- ⚠️ fix one unrelated issue with broken site reporting

### CODE REVIEW

File: broken-site-reporting/tests.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
                 "expectReportURLPrefix": "https://improving.duckduckgo.com/t/epbf",
                 "expectReportURLParams": [
                     {"name": "category", "value": "paywall"},
-                    {"name": "siteUrl", "value": "https%3A%2F%2Fsub.example.test%2F"},
+                    {"name": "siteUrl", "value": "https%3A%2F%2Fsub.example.test%2Fpath%2Fto%2Fthing.html"},
```
Comment (kdzwinel) [inline broken-site-reporting/tests.json:37]: this is unrelated to https changes, but I found it while updating tests, but full path of the site should be included in the report

File: broken-site-reporting/README.md [hunk 2]
```diff
@@ -52,5 +52,5 @@ for $testSet in test.json
     
     if $test.expectReportURLParams
         for $param in $test.expectReportURLParams
-            expect($url.contains($param.name + '=' + $param.value))
+            expect($url.matchesRegex(/[?&] + $param.name + '=' + $param.value + [&$]/))
```
Comment (kdzwinel) [inline broken-site-reporting/README.md:55]: this is a bug that I made in extension's integration of the tests, doing simple `.contains` is not good enough, params should be checked with left and right boundary

File: https-upgrades/README.md [hunk 3]
```diff
@@ -53,4 +54,21 @@ for $testSet in test.json
     )
 
     expect($upgradedRequestURL === $test.expectURL)
-```
\ No newline at end of file
+```
+
+## Bloom filter sources
```
Comment (kdzwinel) [inline https-upgrades/README.md:59]: bloom filter files are still not readable, but now we can include a source for them here

**Review by marcosholgado — APPROVED** [review]
Nice job! 👍

**Pull request merged**